runtime/src/main/codestarts/quarkus/openapi-generator-codestart/java/README.tpl.qute.md

```diff
@@ -0,0 +1,7 @@
+{#include readme-header /}
+
+If you do not have added the `io.quarkus:quarkus-rest-client-jackson` extension in your project, add it first:
+
+```bash
+quarkus ext add io.quarkus:quarkus-rest-client-jackson
+```
```

Comment:
Maybe we should mention the reactive client as well.

Reply:
Solved! @hbelmiro